tracer/src/Datadog.Trace/Telemetry/Collectors/MetricsTelemetryCollector.cs

```diff
@@ -37,12 +38,16 @@ public void Record(PublicApiUsage publicApi)
     public MetricResults GetMetrics()
     {
         var buffer = Interlocked.Exchange(ref _buffer, _reserveBuffer);
+        var now = DateTimeOffset.UtcNow;
+        var interval = (now - _lastRecording).TotalSeconds;
+        _lastRecording = now;
```

Comment:
We certainly don't care but as I don't exactly know the impact I'd rather ask. In the case where we drop the metrics (cf #4354), it is the good thing to do to record the now as `_lastRecording`?

Reply:
> it is the good thing to do to record the now as _lastRecording

This whole "interval" thing still isn't clear in my mind, so I've removed the commit for now and will follow up on it again in a separate PR